tsl/test/sql/chunk_utils_internal.sql

```diff
@@ -267,6 +267,14 @@ SELECT  _timescaledb_internal.drop_chunk( :'CHNAME1');
 SELECT * from test1.hyper1 ORDER BY 1;
 SELECT * FROM hyper1_cagg ORDER BY 1;
 
+-- check that dropping cagg triggers OSM callback
+SELECT ts_setup_osm_hook();
+BEGIN;
+DROP MATERIALIZED VIEW hyper1_cagg CASCADE;
+DROP TABLE test1.hyper1;
+ROLLBACK;
+SELECT ts_undo_osm_hook();
+
```

Comment:
We should have 1 more test. Expect 2 calls to hooks : 1 for mat hypertable and 1 for the hypertable itself
BEGIN;
DROP TABLE test1.hyper1 CASCADE;
ROLLBACK;

Reply:
Added